docs/getting-started/user-guides/using-paymetheus.md

```diff
@@ -60,9 +62,8 @@ to use the address. This means that the pool can vote on your behalf and you can
 using your own wallet if the pool stops working.  
 
 It DOES NOT give the pool access to your funds. All you are doing is granting voting
-rights to the wallet. This is why you provide a ticket address so the network knows
-where to send your funds to after the ticket votes/expires. The pool does not touch
-your funds.  
+rights to the pool. The pool does not touch your funds. This is very important since
+Paymetheus is unable to vote.
```

Comment:
While this is very important, i'm not sure how it is related to the pool being unable to steal funds.